extension/src/scripts/parsers.js

```diff
@@ -40,7 +40,7 @@ export function getDisplayTransaction(
 }
 
 // DEPRECATE
-export const parseTx = (signMessage, displayedProperties) => {
+export const parseTx = (signMessage, displayedProperties, network) => {
```

Comment:
is this even used? there is a DEPRECATE label

Reply:
Good question. I was thinking the same, but just in case updated it. I can open an issue about this & do it in another PR.

Or should I do this here as well?